### PR TITLE
Updated Readme and Installation instructions

### PR DESCRIPTION
**Pull request by Trilarion**
This is still work in progress. Visit [the branch](https://github.com/Trilarion/qpixel/tree/updated_readme_and_installation_instructions#table-of-contents) to see how it looks.

- updated the README (moved the installation instructions, which were a bit lengthy to their own file INSTALLATION.md)
- added section "Features" which describes what the software is able to do (could probably still be extended a bit)
- added a screenshot of the software to the README as kind of advertisement (used photography community landing page with questions sorted by score)

Still to do:

- finalize code style guidelines for contribution guidelines (depends on [What (where) are the current code standards and commit guidelines?](https://collab.codidact.org/posts/285600))
- actively compare installation instructions with [How to set up a new (development) instance of Codidact/QPixel ](https://collab.codidact.org/posts/280451/280452#answer-280452) and refer there to the installations instructions, update installation instructions
- feedback on listed features (@cellio What would you say are the main features of the software powering Codidact?)

Comment:
- if possible the number of folders and files in the root folder of the Git repository should be reduced in order to have more of the README file on screen for Github visitors so they get more drawn in by the nice screenshot, the features and the remaining stuff (however only if it's not interfering with the programming requirements).

### CODE REVIEW

**Comment by cellio** [comment]
Thank you for working on this!

For the screenshot, could you use a more active community, like maybe Software Dev?  Photography doesn't give the best first impression... thanks.

**Comment by cellio** [comment]
Re listed features:

- For customizations, consider linking to https://meta.codidact.com/posts/280722/280723#answer-280723?
- "Voting on content" -- maybe say a little more, like "Voting on content, and sorting based on a formula that accounts for controversy"?  (Everybody has voting, but we do it a little differently.)
- Maybe add something about abilities based on activity?  (My usual example is that you gain the ability to edit by having your suggestions accepted.)  This is in contrast to abilities based on reputation/karma/etc.

**Comment by Trilarion** [comment]
I added the suggestions by Monica to the "Features" section and made an example screenshot of Software Dev community as a showcase for QPixel. Combined the installation guides (the existing README and the guide on collab.CD). Still waiting on feedback on code standards.

**Comment by sau226** [comment]
Readme changes look good - we should probably move the install docs out to a proper developer docs portal we can keep all the 'official' dev docs inside, but a separate file will do for now

**Comment by Trilarion** [comment]
I took the code standards from the old core project (JavaScript, CSS, HTML, commit messages) which probably still apply and combined them in a single document "CODE-STANDARDS.md" and referred to them instead of the links to the wiki of the archived project as done currently in the CONTRIBUTING guidelines. I think it makes sense to keep code standards and code in the same location. Code standards (especially for Ruby (on Rails)) can be extended later. See also: https://collab.codidact.org/posts/285600

That concludes this PR and it's therefore ready for reviewing and potential merge.

Summary of changes:

- reworked contribution guidelines
- added code standards file
- reworked installation instructions and put into separate file
- reworked readme and added screenshot and feature list
- slightly changed screenshots used in installation instructions

**Comment by cellio** [comment]
Looks good to me!

In Contributing, in the "looking for something to do" part, should we also link to the roadmap project? https://github.com/codidact/qpixel/projects/3  While everything's important eventually and we won't turn away a fix that's not on the roadmap, if people are interested in working on roadmap items in particular, that's helpful.

**Comment by Trilarion** [comment]
@cellio I included a link to the roadmap. From my side this PR is now ready.

It clashes partly with #768  Whatever is merged first will result in the other one needing to be slightly adapted.

**Comment by cellio** [comment]
This looks good to me.  @luap42 , opinions?

**Comment by cellio** [comment]
@sau226 or @Trilarion , can one of you resolve the merge conflict on README here?  Since this PR does not change any code and it's been waiting for a while, I'm comfortable saying that it's had enough review and can merge -- except for those conflicts.  Thanks, and sorry for the delay.

**Comment by Trilarion** [comment]
No problem, I should be able to resolve the conflicts.

**Comment by Trilarion** [comment]
Conflict solved. sau226 had modified the installation instructions but I had moved them to their own file. I re-inserted the modifications in an additional commit. You can review and then merge.

File: README.md [hunk 1]
```diff
@@ -19,173 +19,51 @@
...
-
-You'll also need to fill in details for the Redis connection. If you've followed these instructions, the sample file
-should already contain the correct values for you, but if you've customised your setup you'll need to correct them.
+## Features
 
 ### 6. Set up QPixel
 Install gems:
+=======
+## Features
+>>>>>>> moved installation instructions to separate file, added screenshot to readme
```
Comment (cellio) [inline README.md:34]: Is this leftover merge crud?

**Review by cellio — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Trilarion) [reply README.md]: Yes, has been fixed now.

**Pull request merged**